l10n_it_fiscalcode/wizard/compute_fc.py

```diff
@@ -155,20 +141,20 @@ def compute_fc(self):
         for f in self:
             if (not f.fiscalcode_surname or not f.fiscalcode_firstname or
                     not f.birth_date or not f.birth_city or not f.sex):
-                raise except_orm(
-                    _('Error'), ('One or more fields are missing'))
+                raise UserError('One or more fields are missing')
             nat_code = self._get_national_code(
                 f.birth_city.name, f.birth_province.name, f.birth_date)
             if not nat_code:
-                raise except_orm(_('Error'), _('National code is missing'))
+                raise UserError(_('National code is missing'))
             birth_date = datetime.datetime.strptime(f.birth_date, "%Y-%m-%d")
-            CF = build(f.fiscalcode_surname, f.fiscalcode_firstname,
-                       birth_date, f.sex, nat_code)
-            if partner.fiscalcode and partner.fiscalcode != CF:
-                raise except_orm(_('Error'), (
-                    'Existing fiscal code %s is different from the computed'
-                    ' one (%s). If you want to use the computed one, remove'
-                    ' the existing one') % (partner.fiscalcode, CF))
-            partner.fiscalcode = CF
+            c_f = build(f.fiscalcode_surname, f.fiscalcode_firstname,
+                        birth_date, f.sex, nat_code)
+            if partner.fiscalcode and partner.fiscalcode != c_f:
+                raise UserError(_(
+                    'Existing fiscal code {partner_fiscalcode} is different '
```

Comment:
@andrea4ever, as I wrote above: I wouldn't change the message error otherwise you also have to change its translation.
Did you check this translation still work?